test/python/scheduler/test_basic_scheduler.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test cases for the pulse scheduler passes."""
+
+from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
+from qiskit.scheduler.basic_scheduler import basic_schedule
+
+from qiskit.test.mock import FakeOpenPulse2Q
+from qiskit.test import QiskitTestCase
+
+
+class TestBasicSchedule(QiskitTestCase):
```

Comment:
I think tests will have to be extended, but in these situations, it is often unclear about how to write tests which are not just comparing against a ground-truth. It may just be the case that we have to wait until the pulse simulator is ready, at which case we can compare the performance of the circuit vs. pulse sim and check the results are equivalent.

Reply:
The schedule is derived from a quantum circuit which implements a unitary. The schedule should implement approximately the same unitary on the pulse simulator (provided it is properly calibrated), allowing a comparison to be made.

Reply:
yes but the greedy and lazy schedules would both pass this test. how do you distinguish them?

Reply:
For these unit tests here I just want to see that the scheduler schedules things at the expected times. An integrated test to see that a cmd def, scheduler and simulator can faithfully replicate the outcome of a circuit would be great in the future, I think

Reply:
We would likely need a combination of logical equivalence with circuit tests and also ground-truth tests.